src/App.scss

```diff
@@ -2,36 +2,123 @@ iframe {
   display: none;
 }
 
+.title {
+  font-size: 50px;
+}
+
+.containerForApp {
+  width: 100%;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.App {
+  border: 2px black solid;
+  display: flex;
+  flex-direction: column;
+  max-width: 75%;
+  align-items: center;
+}
+
+.formUsers {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  gap: 20px;
+
+  .field {
+    font-family: "Franklin Gothic Medium", Arial, sans-serif;
+    font-size: 30px;
+  
+    &-form {
```

Comment:
such a detailed ampersand overuse may be not needed as it makes code harder to read